src/server.rs

```diff
@@ -584,7 +584,6 @@ fn collect_changes(
 
             for &(component_rule, storage) in &replicated_archetype.components {
                 let (component_id, component_fns, rule_fns) = registry.get(component_rule.fns_id);
-                let send_mutations = component_rule.send_rate.send_mutations(server_tick);
```

Comment:
Removing this makes the code more fragile, because adding a ReplicationMode will not be handled automatically.

Reply:
I don't plan adding more modes (prioritization will be a per-client thing), this is why I removed the method. Having it for 2 variants would be a bit strange.